.github/workflows/ci.yml

```diff
@@ -7,7 +7,7 @@ on:
     branches: [develop]
 env:
     REGISTRY: ghcr.io
-    IMAGE_NAME: twsearle/orca-jedi/ci-almalinux9:feature-use-personal-ghcr-container
+    IMAGE_NAME: twsearle/orca-jedi/ci-almalinux9:v1.2.0
```

Comment:
There is a version bump in this PR, are you not planning to use 1.3.0 or is this a chicken-and-egg problem?

Reply:
So this is the the chicken and egg problem I am still figuring out. I actually realised I could combine this PR with #107 but not until after I had made it!

In this PR I update the container specification to v1.3.0 (containing atlas 0.38.1) but I keep the ci pegged at atlas 0.37.0 so that it still works with that version and will pass. At #107 I update the tests in orca-jedi for validity with 0.38.1, and I update to use the new container created by this PR (when I pushed the tag for v1.3.0)).

This is tricky to explain in text and I am happy to have a call if it helps! As I say, I think I could have combined the two PRs, but I did it this way because I was still understanding this stuff and wanted to make it clear that this PR only affects the container build, not orca-jedi, orca-jedi compatibility with atlas, or anything real - only #107 needs to be coordinated with the rest of mo-bundle changes.

Reply:
As for the reason for this specific line - I figured out that I can make tags even on my PRs - so the container in use on the previous PR should have originally read v1.2.0 as it was compatible with the VERSION of orca-jedi at v1.2.0.